.github/workflows/base.yml

```diff
@@ -69,11 +69,17 @@ jobs:
           python-version: 3.9
           architecture: x64
 
+      # Setup Qt libraries for mayavi pyqt backend used in demo example
+      - name: Setup Qt libraries
+        uses: tlambert03/setup-qt-libs@v1
+
       - name: Install noxfile requirements
         run: pip install -r noxfile-requirements.txt
 
       - name: Run nox session ${{ matrix.nox_session.session }}
-        run: nox -s "${{ matrix.nox_session.session }}"
+        uses: aganders3/headless-gui@v1
```

Comment:
Can you please add a comment above or on the side of this line explaining why this is mandatory ?